src/components/FaqSection.tsx

```diff
@@ -0,0 +1,280 @@
+"use client";
+import { Box, Button, Typography, colors, styled } from "@mui/material";
+import ExpandMoreIcon from "@mui/icons-material/ExpandMore";
+import Link from "next/link";
```

Comment:
For Link component can use from MUI krub
We already set the next Link to be the default component for mui Link so we can use mui props when use the Link like `sx`